src/api/routes/stats.ts

```diff
@@ -0,0 +1,25 @@
+import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox';
+import { FastifyPluginAsync, FastifyPluginCallback } from 'fastify';
+import { Server } from 'http';
+
+const IndexRoutes: FastifyPluginCallback<Record<never, never>, Server, TypeBoxTypeProvider> = (
+  fastify,
+  options,
+  done
+) => {
+  fastify.get('/stats/inscriptions', async (request, reply) => {
+    const inscriptions = await fastify.db.getInscriptionCountPerBlock();
```

Comment:
You need to be able to send a block window so we can limit results e.g. `from_block` `to_block`